packages/kas/.eslintrc.js

```diff
@@ -11,5 +11,23 @@ module.exports = {
                 includeTypes: true,
             },
         ],
+        // To avoid circular dependencies,
+        // we only allow imports from specific Perseus packages.
+        // This is overly strict to prevent accidental interdependencies,
+        // so feel free to update if you know what you're doing.
+        "no-restricted-imports": [
+            "error",
+            "@khanacademy/kas",
```

Comment:
Basically for each of these I added every Perseus package, ran eslint, and removed packages that were already being used.